terraform/modules/database/main.tf

```diff
@@ -71,8 +71,10 @@ module "rds" {
 
   # Enhanced monitoring
   performance_insights_enabled = true
+  create_monitoring_role = true
+  monitoring_role_name = "${var.project}-${var.environment}-rds-monitoring-role"
+  monitoring_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.project}-${var.environment}-rds-monitoring-role"
```

Comment:
The ARN is not required when the previous two parameters are supplied. It will create a role and then use that ARN.

Reply:
I thought so too, it actually errors out when you don't provide it

Reply:
What sort of error?

Reply:
>InvalidParameterCombination: A MonitoringRoleARN value is required if you specify a MonitoringInterval value other than 0.

Feels like some parameter validation enforcement.

Reply:
Yeah let's bump that version up!